src/NuGet.Clients/PackageManagement.PowerShellCmdlets/Model/PowerShellInstalledPackage.cs

```diff
@@ -25,21 +23,32 @@ public class PowerShellInstalledPackage : PowerShellPackage
         /// <summary>
         /// Get the view of installed packages. Use for Get-Package command.
         /// </summary>
-        internal static List<PowerShellInstalledPackage> GetPowerShellPackageView(Dictionary<NuGetProject, IEnumerable<Packaging.PackageReference>> dictionary,
-                                                                                  ISolutionManager SolutionManager, Configuration.ISettings settings)
+        internal static List<PowerShellInstalledPackage> GetPowerShellPackageView(
+            Dictionary<NuGetProject, IEnumerable<PackageReference>> dictionary,
+            ISolutionManager SolutionManager,
+            Configuration.ISettings settings)
         {
             var views = new List<PowerShellInstalledPackage>();
 
             foreach (var entry in dictionary)
             {
                 var nugetProject = entry.Key;
+                var projectName = entry.Key.GetMetadata<string>(NuGetProjectMetadataKeys.Name);
 
-                string packageFolder = null;
                 FolderNuGetProject packageFolderProject = null;
+                VersionFolderPathResolver pathResolver = null;
 
-                if (nugetProject is BuildIntegratedNuGetProject)
+                // Build a project-specific strategy for resolving a package .nupkg path.
+                if (nugetProject is BuildIntegratedNuGetProject ||
+                    nugetProject is ProjectKNuGetProjectBase) // This is technically incorrect for DNX projects,
```

Comment:
These two both share a common interface, it could be used to check, but this is also fine.